src/tests/Microsoft.PowerFx.Connectors.Tests/BasicRestTests.cs

```diff
@@ -83,7 +83,10 @@ public async void ValidateHttpCalls(int i /* used for debugging */, string fxQue
         // Allow side-effects for executing behavior functions (any POST)
         private static readonly ParserOptions _optionsPost = new ()
         {
-            AllowsSideEffects = true
+            AllowsSideEffects = true,
+            
+            // Decimal TODO: why is this needed?  What about decimal doesn't work here?
+            NumberIsFloat = true
```

Comment:
Resolve TODO

Reply:
Leaving this in place for now.  The root issue is coercion that is being handled elsewhere.  When the next phase of Decimal is put in, I'll scan again for Decimal TODO, and can clean this up then.